<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="agreement">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО Родителя (Законного представителя):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="perantName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="childName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата Рождения:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="childBothDay">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Запрос со слов родителя (Законного представителя):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="perantAsk">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тема консультации:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="konsTheam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата консультации:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="KonsDate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дополнительная информация:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moreInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showAgreementBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотреть согласие</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
